cli/select.go

```diff
@@ -0,0 +1,110 @@
+/*
+ * Warp (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cli
+
+import (
+	"github.com/minio/cli"
+	"github.com/minio/minio-go/v6"
+	"github.com/minio/warp/pkg/bench"
+)
+
+var (
+	selectFlags = []cli.Flag{
+		cli.IntFlag{
+			Name:  "objects",
+			Value: 2500,
+			Usage: "Number of objects to upload.",
+		},
+		cli.StringFlag{
+			Name:  "obj.size",
+			Value: "10MiB",
+			Usage: "Size of each generated object. Can be a number or 10KiB/MiB/GiB. All sizes are base 2 binary.",
+		},
+		cli.StringFlag{
+			Name:  "query",
+			Value: "select * from s3object",
```

Comment:
I wanted to have it do something "more" for the first version, but let's add it and refine later.

But this is a bit too easy to "cheat", but the complexities easily explode.

Let's keep this for now.